docs/GraFx-Developers/connectors/authorization-for-connectors/index.md

```diff
@@ -0,0 +1,287 @@
+
+## Authorization Limitations
+
+While the Connector framework supports the most common authorization schemas, it's important to note that completely customizable authorization is not possible. If your specific authorization requirements don't align with the four provided types, you may need to develop a workaround solution.
+
+For example, services like GraFx Media and GraFx Platform don't meet the requirements of any of the supported authorization types. In such cases, you would need to implement an intermediary service that adapts the authorization workflow to work within the Connector framework.
+
+## Define Supported Authorization Types
+Before implementing authorization, you must define the supported authorization types in your `package.json` file. This ensures consistency between your CLI commands and your package configuration.
+
+To do so, add or modify the `supportedAuth` property you `package.json` file with the types of authentication your Connector supports.
+
+Example of a `package.json` supporting Static Header Key and OAuth Authorization Code:
+```json
+{
+...
+  "config": {
+    ...
+    "supportedAuth": ["staticKey", "oAuth2AuthorizationCode"]
+  }
+}
+```
+
+## Implement Authorization
+To implement authorization in a Connector, you need a published Connector and the [Connector CLI]() tool. Use the following command:
+
+```bash
+connector-cli set-auth \
+    --connectorId <connector-id> \
+    -e <environment> \
+    -b <base-url> \
+    -au <auth-usage> \
+    -at <auth-type> \
+    --auth-data-file <path-to-auth-file>
+```
+
+<br/>
+
+!!! note "Authorization is Optional"
+
+	  Authorization is optional for Connectors. By default, no authorization is used. However, once authorization is set, there is currently no method to remove it via the Connector CLI.
+
+### Connector ID (`--connectorId`)
+
+This argument is the ID of your connector published in your GraFx environment.
+
+To retrieve all Connectors to get lookup your ID, use the following API endpoint (with proper authorization header):
+
+```bash
+GET https://{environment}.chili-publish.online/grafx/api/experimental/environment/{environment}/connectors
+```
+
+### Environment (`-e`)
+
+This argument expects the environment name that your Connector is published in.
+
+### Base URL (`-b`)
+
+This argument expects the base URL for GraFx API calls, which will be in either of the following formats:
+
+- Production Environments
+```
+https://{environment}.chili-publish.online/grafx
+```
+
+- Sandbox Environment
+```
+https://{environment}.chili-publish-sandbox.online/grafx
+```
+
+!!! tip "Matching Environment"
+
+    Ensure that the environment in your base URL matches the environment passed via the `-e` argument.
+
+
+### Authorization Usage (`-au`)
+
+This argument expects one of two value that will specify the usage scenario for the authorization.
```

Comment:
Grammar, change to "This argument expects one of two values that will specify the usage scenario for the authorization."